modules/data.atmosphere/R/download.NOAA_GEFS.R

```diff
@@ -35,7 +35,8 @@
 ##' @param username username from pecan workflow 
 ##' @param overwrite logical. Download a fresh version even if a local file with the same name already exists?
 ##' @param downscale logical, assumed True. Indicated whether data should be downscaled to hourly
-##' @export
+##' @param ...  Additional optional parameters
+##'  @export
```

Comment:
same as other PR (which has duplicate code) you need to fix the stray space here, add export to the helper functions, and make document